### DISCUSSION TITLE
is the project abandoned ?

### DISCUSSION THREAD

**Discussion by noor-tg** [emacs-ng/emacs-ng#439] [Q&A] [answered]
there is no update from 7/4/2022.

**Comment by brotzeit** [comment, marked as answer]
The problem is that there are not enough experienced people maintaining the project. I was hoping we could encourage more people to help when the project provides new features that users are interested in. However we obviously didn't get to that point. I would gladly give it another try, but I'm not able to fix the more complicated issues myself or implement cool new stuff(at least not at a level that would make a difference). Maybe some users will continue the work that was started. I would join them ;)

However it seems to me that the lack of maintainers and contributors in the emacs community is an issue that rather gets worse than better. It's cool that there are so many new users using doom-emacs etc. but we also need people willing to improve emacs.
So to be honest, I doubt that will happen. But I would be very happy if I was wrong.
